giftless/transfer/basic_streaming.py

```diff
@@ -144,9 +149,16 @@ def download(self, organization: str, repo: str, oid: str, size: int,
 
         else:
             download_url = ObjectsView.get_storage_url('get', organization, repo, oid)
+            preauth_url = self._preauth_url(download_url, organization, repo, actions={'read'}, oid=oid)
+
+            params = {'filename': extra.get('filename') if extra else None}
```

Comment:
In terms of style, I try to avoid the short-form `if` unless it's really easy to read. 

In general, this whole block of code and the `filename=` parameter can be skipped unless `extra and 'filename' in extra`. I'd write this whole block under condition.

Reply:
Thanks for the great feedback.  I'm hoping I've resolved this now.